ee/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "longname": "Exact Editions",
+  "name": "ee",
+  "describe": "Recognizes the accesses to the platform Exact Editions",
+  "contact": "Sean Duffy",
+  "pkb": false,
+  "docurl": "http://analyses.ezpaarse.org/platforms/615512befd8f6768387e62b7",
+  "domains": [
+    "reader.exacteditions.com.ucark.idm.oclc.org",
```

Comment:
This url seems to be proxy based, if so, it is not needed here since the url without proxy is present in the manifest.